context/app/test_routes_errors.py

```diff
@@ -8,8 +8,10 @@
 
 @pytest.fixture
 def client():
-    app = create_app()
-    app.config['TESTING'] = True
+    app = create_app(testing=True)
+    # gitignored instance/app.conf should not be read during tests:
+    # We should just get the default config.
+    assert 'IS_MOCK' not in app.config
```

Comment:
ie, app.conf might have turned on IS_MOCK: We want to make sure we're trying to make HTTP requests... and the mocking is *inside* the test.